src/components/Event/Event.tsx

```diff
@@ -140,10 +128,10 @@ const Event = (props: EventProps) => {
         buttonColor={"red"}
         buttonUrl={"test"}
```

Comment:
I think we should add this back in so the user can set the link of the page that the button should go to